src/pkg/projects/netcoreapp/sfx/theme/1046/bundle.wxl

```diff
@@ -1,74 +1,74 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looks like the branch has checked-in CRLF rather than LF. I'll add a quick commit to your branch to clean it up.